x-pack/plugins/observability_solution/synthetics/server/synthetics_service/project_monitor/normalizers/common_fields.ts

```diff
@@ -145,8 +142,27 @@ export const getCustomHeartbeatId = (
   return `${monitor.id}-${projectId}-${namespace}`;
 };
 
-export const getMonitorSchedule = (schedule: number | string | MonitorFields['schedule']) => {
+export const getMonitorSchedule = (
+  schedule: number | string | MonitorFields['schedule'],
+  defaultValue?: MonitorFields['schedule']
+) => {
+  if (!schedule && defaultValue) {
+    return defaultValue;
+  }
   if (typeof schedule === 'number' || typeof schedule === 'string') {
+    if (typeof schedule === 'number') {
+      return {
+        number: `${schedule}`,
```

Comment:
```suggestion
        number: String(schedule),
```

Can this be simpler?